test/utils/helper.ts

```diff
@@ -0,0 +1,14 @@
+import { TextDocument } from 'vscode-languageserver-textdocument';
+import * as path from 'path';
+import { promises as fs } from 'fs';
+
+
+export async function getDoc(filename: string): Promise<TextDocument> {
+  const yaml = await fs.readFile(
```

Comment:
If this function can be used for things other than yaml (since `yaml` was removed from the function name, I guess maybe that's the case), then maybe change this var name.